### DISCUSSION TITLE
EACCESS error

### DISCUSSION THREAD

**Discussion by Nichneo** [community/community#7695] [Copilot Conversations] [answered]
Hello,
I'm getting this error after installing the extension:
![image](https://user-images.githubusercontent.com/49399608/141681615-ee88e29a-19be-4ca2-b457-da3826a0acd8.png)
I guess it's something related to Proxy, but I have both HTTP and HTTPS_PROXY variables defined and I have no issues using other process (like installing npm packages, using Git, etc..).
Any suggestion?
Many thanks!

**Comment by davecheney** [comment, marked as answer]
I’m sorry at the moment copilot does not support http proxies.

**Reply by Nichneo** [reply]
Hi @davecheney, thanks for the prompt feedback, then if you'd like to you can free up my pilot license for somebody else, as I only develop through proxy. I can re-join the pilot once you'll need somebody to test proxy connection then.
